packages/flutter/lib/src/widgets/table.dart

```diff
@@ -348,39 +348,44 @@ class _TableElement extends RenderObjectElement {
   @override
   RenderTable get renderObject => super.renderObject as RenderTable;
 
-  // This class ignores the child's slot entirely.
-  // Instead of doing incremental updates to the child list, it replaces the entire list each frame.
-
   List<_TableElementRow> _children = const<_TableElementRow>[];
 
   @override
   void mount(Element? parent, dynamic newSlot) {
     super.mount(parent, newSlot);
-    _children = widget.children.map<_TableElementRow>((TableRow row) {
-      return _TableElementRow(
-        key: row.key,
-        children: row.children!.map<Element>((Widget child) {
-          assert(child != null);
-          return inflateWidget(child, null);
-        }).toList(growable: false),
-      );
-    }).toList(growable: false);
-    _updateRenderObjectChildren();
```

Comment:
After the slot is introduced, we can't add RO children here uniformly, we can only add children to RO through `void insertRenderObjectChild`. 

This is exactly why we need to refactor the render. I don't want to modify it on a large scale, but we have to. `RenderTable` is really different from other similar multi-child RO implementations. I don't know if the author at the time had a special purpose. My modification is to make its implementation not so special, and then the problem is solved naturally.

Reply:
@Hixie